_harp/academy/_about.md

```diff
@@ -2,4 +2,18 @@
 
 <img src="/assets/fac-logo.png" class="fac-logo">
 
-Founders & Coders runs an intensive full-time, 16-week course in full-stack software development using JavaScript and Node.js. Application is highly-selective and we take just 16 students per cohort. After graduation, we expect all of our students to help us deliver the course to the next cohort and to contribute to our various community projects.
+Founders & Coders has created a new model for education.
+
+We run a full-time, 16-week course in full-stack software development using JavaScript and Node.js. 46/47 of our recent graduates are working as software developers and around 40% of our graduates are female.
+
+We aim to give students the best possible training in software development. In return we ask you to help us deliver subsequent courses and to contribute to our various community projects.
+
+We also ask you to contribute to the academy's sustainability in one of three ways:
+
+- Join our community - earning a living while learning and working on interesting projects with our senior developers.
+
+- Let us help you find full time employment with one of our hiring partners who will provide us with a referral fee.
+
+- Donate 10% of your salary for your first year of employment.
```

Comment:
I've just had a cracking idea, in the interest of fairness: what if we took 50% of people's salary above the median salary for their age group? So, let's say for a 25 year-old, the median salary is probably about £20k; if after FAC they get a job paying £30k, then they pay £5k back to FAC.

Now, that might sound like a lot of money, but think about what the median salary means - it means that half of people earn less than that. So while they're paying £5k to FAC, they're still earning £5k more than half of the people in their age group. 

Obviously everyone's going to baulk at these numbers, but try to think in terms of a system that takes into account the value that FAC is creating for you.